packages/grid/x-data-grid/src/models/events/gridEventLookup.ts

```diff
@@ -441,7 +441,6 @@ export interface GridEventLookup
   sortedRowsSet: {};
   /**
    * Fired when the expansion of a row is changed. Called with a [[GridGroupNode]] object.
-   * @ignore - do not document.
    */
   rowExpansionChange: { params: GridGroupNode };
```

Comment:
> I wanted to add more things like isExpanded and row to the params of event as they could be useful in certain customizations, but this might be a breaking change for some users

You can add another param to the event if it can be useful:
```ts
rowExpansionChange: { node: GridGroupNode,  params: { isExpanded: boolean } }
```